chromium_src/chrome/browser/android/preferences/pref_service_bridge.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 3.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/android/chrome_jni_headers/chrome/jni/PrefServiceBridge_jni.h"
+#define JNI_PrefServiceBridge_GetAboutVersionStrings \
+  JNI_PrefServiceBridge_GetAboutVersionStrings_ChromiumImpl
+#include "../../../../../chrome/browser/android/preferences/pref_service_bridge.cc"
+#undef JNI_PrefServiceBridge_GetAboutVersionStrings
+
+static ScopedJavaLocalRef<jobject> JNI_PrefServiceBridge_GetAboutVersionStrings(
+    JNIEnv* env,
+    const JavaParamRef<jobject>& obj) {
+  JNI_PrefServiceBridge_GetAboutVersionStrings_ChromiumImpl(env, obj);
```

Comment:
Calling the Chromium implementation here to avoid warning about unused static function.

Reply:
if you don't need the original I think you can just skip renaming and just undef at the end

Reply:
OK, I feel like it was having problems with that due to it being a static non-member function, but I can try again.

Reply:
what was the exact error?

Reply:
```/home/emerick/work/brave-browser/src/brave/chromium_src/chrome/browser/android/preferences/pref_service_bridge.cc:10:36: error: redefinition of 'JNI_PrefServiceBridge_GetAboutVersionStrings'
static ScopedJavaLocalRef<jobject> JNI_PrefServiceBridge_GetAboutVersionStrings(
                                   ^
../../third_party/gvr-android-sdk/src/libraries/headers/../../../../../chrome/browser/android/preferences/pref_service_bridge.cc:1028:36: note: previous definition is here
static ScopedJavaLocalRef<jobject> JNI_PrefServiceBridge_GetAboutVersionStrings(
                                   ^
/home/emerick/work/brave-browser/src/brave/chromium_src/chrome/browser/android/preferences/pref_service_bridge.cc:13:3: error: use of undeclared identifier 'JNI_PrefServiceBridge_GetAboutVersionStrings_ChromiumImpl'; did you mean 'JNI_PrefServiceBridge_GetAboutVersionStrings'?
  JNI_PrefServiceBridge_GetAboutVersionStrings_ChromiumImpl(env, obj);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  JNI_PrefServiceBridge_GetAboutVersionStrings
/home/emerick/work/brave-browser/src/brave/chromium_src/chrome/browser/android/preferences/pref_service_bridge.cc:10:36: note: 'JNI_PrefServiceBridge_GetAboutVersionStrings' declared here
static ScopedJavaLocalRef<jobject> JNI_PrefServiceBridge_GetAboutVersionStrings(
                                   ^
2 errors generated.
```

Possibly complicated by the fact that `PrefServiceBridge_jni.h` calls `JNI_PrefServiceBridge_GetAboutVersionStrings`...

Reply:
I'll just go with calling the original function at the top as I have it now, since it doesn't do any harm and there doesn't seem to be an easy or obvious way to avoid it.